src/app/models/student.ts

```diff
@@ -1,6 +1,6 @@
 export interface Student {
     avatar?: string;
-    classId: number
+    classId: number;
     classe?: string;
```

Comment:
classe typo?

Reply:
Sorry, i dont know who is the owner of this interface, it's not mine

Reply:
ask the owner to fix this

Reply:
or step into a trail of goodness =)